accepted/2020/minimum-os-version/minimum-os-version.md

```diff
@@ -40,124 +40,53 @@ To enable tooling to provide a good experience, assemblies should be annotated w
-}
-
-public enum AvailabilityKind
-{
-  Introduced,
-  Obsoleted,
-  Removed
-}
-```
-
-The platform string is expected to have the same values as used in the `TargetPlatformIdentifier` portion of platform-specific TFMs. Having it in the availability attributes may seem redundant as it will also be burned into assembly attributes for platform-specific reference assemblies. However, explicitly specifying the attribute is useful for platform-agnostic `net5.0` reference assemblies that have methods that are only available on some platforms e.g. due to having platform-specific implementations via bait-and-switch. This may also improve source readability when multi-targeting as it will remove the need to use `#if` around the annotations.
-
-Constants for the platform identifiers could perhaps be exposed on a `PlatformIdentifier` type.
-
-The values declared on this attribute are expected to be accessible by extracting constructor arguments from metadata. However, for readability it’s expected to be applied using subclasses with more specific names. To enforce this, it has an internal constructor and its subclasses are sealed.
-
-The concrete subclasses are `IntroducedAttribute`, `ObsoletedAttribute` and `RemovedAttribute` subclasses, which take of the following form:
-
-```csharp
-public sealed class IntroducedAttribute : PlatformAvailabilityAttribute
-{
-    public IntroducedAttribute(
-        string platform,
-        int majorVersion,
-        int minorVersion = 0,
-        int buildVersion = 0,
-        int revisionVersion = 0,
-        string message = null
-    ) : base(platform, majorVersion, minorVersion, buildVersion, revisionVersion, message) {}
-}
-```
-
-These attributes are used as follows:
-
+Earlier versions of this section proposed a form of these annotation attributes based on the Xamarin.iOS/Xamarin.Mac attributes, but it has been superseded by a further refined and generalized form defined in the [attributes section of the Platform Checks spec](../platform-checks/platform-checks.md#attributes).
 
-```csharp
-[Introduced(PlatformIdentifier.macOS, 10, 2)]
-[Deprecated(PlatformIdentifier.macOS, 10, 9)]
-[Removed(PlatformIdentifier.macOS, 10, 12, 1)]
-public class Foo : NSObject { ... }
-```
-
-The purpose of keeping ‘removed’ APIs and annotating them with `RemovedAttribute` is to allow apps and libraries to continue to use these APIs in fallback paths for older OS versions. This is a pattern that exists on Apple platforms.
-
-If and when C# support target typing for enums, we could add convenience overloads that would improve readability:
+These availability attributes omit the `Deprecated` availability kind from the Xamarin attibutes as it’s redundant, and rename `Introduced` to `Supported` to better reflect their alternate purpose of annotating multi-platform assemblies. They also omit the ability to differentiate availability based on architecture for reduction of scope, thought this could be added if needed in future.
```

Comment:
```suggestion
These availability attributes omit the `Deprecated` availability kind from the Xamarin attributes as it’s redundant, and rename `Introduced` to `Supported` to better reflect their alternate purpose of annotating multi-platform assemblies. They also omit the ability to differentiate availability based on architecture for reduction of scope, though this could be added if needed in future.
```